src/en/superstream/src/eu/kanade/tachiyomi/animeextension/en/superstream/SuperStreamAPI.kt

```diff
@@ -1,4 +1,4 @@
-/* 
+/*
```

Comment:
What is this btw???

Reply:
Must have been a space there and Studio removed it during auto format... I added the license while on my phone cause the cs dude was upset

Reply:
Yeah my github app doesn't display spaces in the diff.
That's why I was confused because the two lines looked identical.
I can see the space removed in the browser.